process/smartContract/builtInFunctions/esdtNFTTransfer.go

```diff
@@ -111,6 +111,9 @@ func (e *esdtNFTTransfer) ProcessBuiltinFunction(
 	if len(vmInput.Arguments) < 4 {
 		return nil, process.ErrInvalidArguments
 	}
+	if e.shardCoordinator.ComputeId(vmInput.RecipientAddr) == core.MetachainShardId {
```

Comment:
will this be backwards compatible after will launch the ESDT/NFTs subsystem?

Reply:
this enters the release before the launch.